src/Processors/AugmentProperties.php

```diff
@@ -113,4 +112,26 @@ public function __invoke(Analysis $analysis)
             }
         }
     }
+
+    /**
+     * @param string $type
+     *
+     * @return string
+     */
+    protected function exractType(string $typeString)
+    {
+        if (strpos($typeString, '|') === false) {
+            return $typeString;
+        }
+
+        $types = explode('|', $typeString);
+        foreach ($types as $type) {
+            $type = trim($type);
+            if ($type !== 'null') {
+                return $type;
+            }
+        }
+
+        return 'UNKNOWN';
```

Comment:
why UNKNOWN? does it have special meaning?

Reply:
It is not in the mapping of known types and triggers the right behavior. I will try to change it to the UNDEFINED constant.